app/views/client_details_long.html

```diff
@@ -279,6 +279,8 @@ <h2 class="govuk-heading-m">What happens next</h2>
             <p class="govuk-body">We'll check your client's benefits status with the Department of Work and Pensions (DWP). </p>
           </div>
 
+          {% if mvp == true %}
```

Comment:
What you can do here is just say `{% if mvp == false %}` and remove the `{% else %}`